.github/workflows/windows.yml

```diff
@@ -421,6 +421,18 @@ jobs:
         env:
           TEST_DEVICE: CPU
 
+      - name: JAX Layer Tests - TF FE
+        if: ${{ always() }}
+        shell: cmd
+        run: |
+          python3 -m pip install -r ${{ env.LAYER_TESTS_INSTALL_DIR }}/requirements.txt
+
+          set PYTHONPATH=${{ env.OPENVINO_REPO }}\tools\mo;${{ env.LAYER_TESTS_INSTALL_DIR }};%PYTHONPATH%
```

Comment:
Do we need `${{ env.OPENVINO_REPO }}\tools\mo` ? It's a deprecated tool.
I hope we should base everything on OVC now

Reply:
Fixed.